pom.xml

```diff
@@ -91,6 +91,12 @@
 			<groupId>org.seleniumhq.selenium</groupId>
 			<artifactId>selenium-java</artifactId>
 			<version>3.141.59</version>
+			<exclusions>
```

Comment:
Regarding [my comment](https://github.com/retest/recheck-web/pull/247#issuecomment-498218476): I was only curious what dependency needs ByteBuddy. I believe we cannot just exclude ByteBuddy here, because then user has to declare it explicitly (at least that's my assumption). So I guess we have to add it *before* Selenium.

Reply:
I'm open to any workaround, but I prefer to write Mock dependency before Selenium.